src/components/ProjectForm.js

```diff
@@ -201,6 +219,74 @@ import { Link } from 'gatsby'
             <p className="help">This field is required</p>
           </div>
           </div>
+          </div>
+		  
+		  <div className="columns is-centered is-multiline">
+          {/* DIVIDER BEFORE PROJECT */}
+          <div className="column is-10 is-centered">
+            <hr />
+            <center>
+            <h4 className="has-text-weight-bold is-size-5 has-text-primary">
+            </h4>
+            </center>
+          </div>
+          </div>
+		  
+		  <div className="columns is-centered is-multiline">
+		  {/* SPONSOR NAME */}
+          <div className="column is-3">
+            <br/>
+            <label className="label is-large">Project Sponsor</label>
```

Comment:
```suggestion
            <label className="label is-medium">Project Sponsor</label>
```
So things line up properly